packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -79,6 +79,7 @@ const injectedPropTypes = {
 export const ownPropTypes = {
   customActivityTypes: PropTypes.object,
   initialActivity: PropTypes.string,
+  isAcknowledgementsDisabled: PropTypes.bool,
```

Comment:
Should we call this `hasAcknowledgementsDisabled`? I feel it's better grammatically, and signals that "Acknowledgements" inside this component are disabled (e.g. `hasNotificationsMuted`)
I've always considered the `is` prefix to be something specific to this component that doesn't need another noun to tell you what it is (e.g. `isOff`, `isMuted`). What do you think?

Reply:
Yea, makes sense!